<launch>
	<node pkg="gsft_control" type="tuning_nominal_node" name="tuning_nominal" output="screen">
		<!-- <rosparam command="load" file="$(find gsft_control)/cfg/firefly.yaml" /> -->
		<remap from="odometry" to="/msf_core/odometry" />
		<remap from="command/motor_speed" to="/fcu/command/direct_motor" />
	</node>
</launch>
